### PR TITLE
Update URL format for Tweet 

### PR DESCRIPTION
**Pull request by Willjobs94**
Update Url format for tweet according to #336

### CODE REVIEW

**Comment by codnee** [comment]
@Willjobs4 URL's shouldn't be hardcoded. Use URLHelper to build them instead.

**Comment by eatskolnikov** [comment]
@Willjobs4 @codnee  tiene un punto

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ empleo-dot-net
 
 ### What is this ?
 
-The porpuse of this project is to bring a "real-in-production project" experiance for all developers interesting in learning ASP.Net MVC 5 and Xamarin.
```
Comment (claudiosanchez) [inline README.md]: Se te coló un cambio al readme que no es parte del URL del Tweet. ¿Dando la milla extra?

**Comment by eatskolnikov** [comment]
@Willjobs94 si quitas el cambio del readme yo lo veo todo bien, dale y avisas para hacer el merge

**Comment by Willjobs94** [comment]
Why should i need to remove the changes on the readme, do we need to create another issue to make the changes ?

**Comment by eatskolnikov** [comment]
@Willjobs94 yeah! por convención es 1 PR = 1 issue, voy a agregar instrucciones en el readme para evitar que este escenario suceda de nuevo.

**Comment by Willjobs94** [comment]
Opps.

**Comment by eatskolnikov** [comment]
@Willjobs94 Oops de mi lado, I'm sowy :v

**Comment by Willjobs94** [comment]
@eatskolnikov I've made the modification according with @codnee's suggestion, I'm just missing a rebase :smile:

**Comment by eatskolnikov** [comment]
@codnee revisa y da plus one si lo ves bien

**Comment by Willjobs94** [comment]
@eatskolnikov  I'm having problems rebasing those commits, so I guess is time to check the new [squash github's feature](https://github.com/blog/2141-squash-your-commits) :wink:

**Comment by claudiosanchez** [comment]
@Willjobs4 Las pruebas están fallando.

**Comment by claudiosanchez** [comment]
`Services\Twitter\TwitterServiceTests.cs(45, 37): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'
Services\Twitter\TwitterServiceTests.cs(74, 37): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'
Services\Twitter\TwitterServiceTests.cs(94, 37): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'
Web\Controllers\JobOpportunityControllerTest.cs(241, 62): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'
Web\Controllers\JobOpportunityControllerTest.cs(266, 62): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'
Web\Controllers\JobOpportunityControllerTest.cs(308, 47): error CS7036: There is no argument given that corresponds to the required formal parameter 'urlHelper' of 'ITwitterService.PostNewJobOpportunity(JobOpportunity, UrlHelper)'`

**Comment by Willjobs94** [comment]
yeah I'm working on it!

**Comment by Willjobs94** [comment]
@eatskolnikov the test have passed, anything else ?

**Comment by codnee** [comment]
Esos keys y secrets no deben estar en Auth.config

**Comment by Willjobs94** [comment]
@codnee, what can I do, delete the file, or what do you propuse ?

**Comment by codnee** [comment]
Supongo que el sitio esta corriendo en Azure Web Apps, asi que esas cosas se pueden agregar en el admin portal. Yo borraria el archivo, pero te recomiendo que esperes confirmación de alguien que este mas familiarizado con el proyecto web.

**Comment by anyeloamt** [comment]
@Willjobs94 Al archivo `Auth.config` no se le debería hacer tracking en Git. El truco está en que Visual Studio incluya el archivo, pero Git no. Así los secrets se mantienen en secreto, valga la redundancia.

[Aquí ](http://www.asp.net/identity/overview/features-api/best-practices-for-deploying-passwords-and-other-sensitive-data-to-aspnet-and-azure) hay un artículo corto que lo explica jevi

**Comment by Willjobs94** [comment]
Thanks @anyeloamt, I'll change it right now.

**Comment by Willjobs94** [comment]
@anyeloamt, @codnee, done!, Thanks for feedback

**Comment by luis-ramirez** [comment]
@Willjobs94 and the rebase?

**Comment by Willjobs94** [comment]
@luis-ramirez, they made some merge to the codebase when I was still working with this PR, after I solved the conflicts and tried to make a rebase, It was taking others commits as well, It was a disaster, I couldn't just take my commits to make the rebase, that's why I made the suggestion to use the new (github's squash feature) [https://github.com/blog/2141-squash-your-commits]. Or you can also help me with that in order to find "el maco".

**Pull request merged**